release-notes/releases-index.json

```diff
@@ -12,7 +12,8 @@
             "support-phase": "preview",
             "eol-date": "2026-05-11",
             "release-type" : "sts",
-            "releases.json": "https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/9.0/releases.json"
+            "releases.json": "https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/9.0/releases.json",
+            "supported-os.json": "https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/9.0/suported-os.json"
```

Comment:
Good catches. Fixed the typos.

I'm waiting on the files to be published to blog storage before merging this PR. I suspect it will be soon.

@leecow